dropbox/oauth.py

```diff
@@ -412,10 +456,19 @@ def __init__(self, consumer_key, consumer_secret, redirect_uri, session,
             user - include user scopes in the grant
             team - include team scopes in the grant
             Note: if this user has never linked the app, include_granted_scopes must be None
+        :param str pkce_method: method for code_challenge generation if using PKCE
```

Comment:
Again I feel we should just use a boolean here:

And the doc string should be more lean toward something like: "PKCE should be only use on client apps which doesn't call your server. It is less secure than non-PKCE flow".